blog/_posts/2018-07-09-dvteam-activated.md

```diff
@@ -0,0 +1,75 @@
+
+### RgSoC Coaches & Supervisor
+During the Google Scholarship, we met several times in real life and decided to apply for RgSoc to be able to keep practicing our coding skills in summer. As we were new in the dev community, we reached out to our Udacity community managers who introduced us to Amanda Cavallaro. She helped us find our first two current RgSoc-coaches: 
+
+*Coach 1: Tobias (React developer)* 
+
+*Coach 2: Tetiana (Ruby developer)*
+
+Skyping with Tobias and Tetiana, gave us the confidence they would have our backs! They not only seem very nice and accessible, but moreover have the attitude and experience needed to guide us through the summer.  
+
+A few weeks later we met our third mentor through the post system in RGSoC.We absolutely loved his passion and diverse knowledge of programming so we both agreed to have him on board!
+
+*Coach 3: Raounak (Full-stack Ruby developer)*
+
+The last RgSoc-coach to join, we met during a recently peer lab, that Violeta organized for Woman Who Code. We really liked her spirits and clarity so just a week before starting the SoC we invited her to be part of it. 
+
+*Coach 4: Cristina (full stack developer)*
+
+We consider ourselves very lucky! As not only do we have gathered a great team of mentors, but also an interesting supervisor Lucas Pinto who has even organized Ruby conferences! 
+
+### Living Style Guide
+The Open Source project we have been selected for is Living Style Guide, which helps developers to create front-end style guides easily with SaSS and Markdown. The project is written by Nico Hagenburger in Ruby. It needs to be rewritten to and tested for a Vanilla Javascript version, which will be our main tasks, beside improving the documentation. 
+
+Summer and future Goals
+We are looking forward to connect with the RgSoc-community, make our first ever contribution to open source (yay!!), hone our skills and learn many more. We are here to prove ourselves that we can do this! And hope to be ready to find a tech job when the summer is over.
+
+
+Our summer will be successful, _if_ we achieve the following:
+
+*July:* plan, learn as much as we can, start contributing to Living Style Guide, connect with the RgSoc-community
+
+*August:* keep working hard and make the best out of the summer!
+
+*September:* finish as much as we can of the Javascript version & test of Living Style Guide
+
+*October:* start a new tech job as jr. front end web developers in Barcelona.
+
+#### DV Team (Violeta & Diana)- Barcelona
+
+Follow us on
+
+##### Medium
+[DV Team](http://www.medium.com/@dvteam)
+
+[Diana](https://medium.com/@diana.vile)
+
+##### Twitter
+[Violeta](http://www.twitter.com/artnerdnet)
+
+##### Github
+[Living Style Guide](https://github.com/livingstyleguide/livingstyleguide)
+[Violeta](http://www.github.com/artnerdnet)
+[Diana](http://www.github.com/dianavile)
+
+# For a great beginning!
+![image](https://media.giphy.com/media/WeJeWpUMxpQmk/giphy.gif)

```

Comment:
Please add credits using image-credits class.

Reply:
not sure if I did it correctly